modules/api_reference/devportal_api_reference.api.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * Hooks provided by the Devportal API Reference module.
+ */
+
+use Drupal\devportal_api_reference\ReferenceInterface;
```

Comment:
Always use FQCN-s in api.php because it ensures that the correct class/interface gets referenced when PHPstorm hooks autocomplete is being used.